src/controllers/search.controller.js

```diff
@@ -0,0 +1,18 @@
+const searchService = require('../services/search.service');
+
+const searcH = async(req,res) => {
+    try{
+        const name = req.params.categories_name;
+
+        if(!name) {
+            throw new Error("Key Error")
+        }
+        const result = await searchService.searcH(name);
+
+        return res.status(201).json({message : result})
```

Comment:
```suggestion
        return res.status(200).json({message : result})
```
해당 api는 db에 data를 추가 및 수정(POST)하는 것이 아닌, 단순 데이터를 받아오는(GET) api입니다. 적절한 status code를 사용 할 수 있게 수정해주세요~